Weather Alert/CoreData/DataController.swift

```diff
@@ -11,18 +11,18 @@ import CoreData
 
 class DataController {
     
-    fileprivate lazy var applicationDocumentsDirectory: URL = {
+    fileprivate lazy var applicationDocumentsDirectory: URL! = {
```

Comment:
In this case the use `!` is good as app shouldn't even start if this setup is incorrect. Also, this setup is handled by the developer and not third party (e.g. API) so it'll most likely to work and we'll know if anything is wrong:)